src/commands/up.rs

```diff
@@ -257,3 +256,33 @@ fn parse_env_var(s: &str) -> Result<(String, String)> {
     }
     Ok((parts[0].to_owned(), parts[1].to_owned()))
 }
+
+fn try_trigger_plugin(trigger_type: &str) -> Result<String> {
```

Comment:
Given the Rust `try_*` convention, I read this as "try to trigger the plugin". Maybe "resolve"?

```suggestion
fn resolve_trigger_plugin(trigger_type: &str) -> Result<String> {
```